doc/en/user/source/extensions/wps/security.rst

```diff
@@ -42,3 +42,37 @@ The roles string must be a simple semicolon separated list:
    :align: center
 
    *The role editor inside the process list page*
+   
+Input limits
+------------
+
+The amount of resources used by a process is normally directly related to the inputs of the process itself.
+The WPS specification is conscious of that, and allows to declare three different type of limits on each process inputs:
+
+* The maximum size of complex inputs, in megabytes
+* The range of acceptable values for numeric values
+* The max multiplicity of repeatable inputs (think of contour extraction, where the number of levels affects the execution time)
+
+Starting with GeoServer 2.7.0, GeoServer allows the administrator to configure said limits, and fail requests that 
```

Comment:
I don't think we need to say the version number - it just makes the documentation look stale over time. Each page here says GeoServer 2.7.0 at the top.